extensions/configuration/servicebindings.yaml

```
@@ -0,0 +1,112 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: servicebindings
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: servicebindings
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  details: |
+    header:
+      - name: spec.controlledBy
+        source: spec.controlledBy
```

```suggestion
      - name: spec.controlledBy
        source: metadata.ownerReferences
```
Please adjust translation and name